apps/api/v2/src/ee/event-types-private-links/services/organization-context.service.ts

```diff
@@ -0,0 +1,36 @@
+import { OrganizationsRepository } from "@/modules/organizations/index/organizations.repository";
+import { TeamsEventTypesRepository } from "@/modules/teams/event-types/teams-event-types.repository";
+import { Injectable, NotFoundException } from "@nestjs/common";
+
+export interface OrganizationEventTypeContext {
```

Comment:
I don't see this interface being used anywhere - do we need it or can we delete it all together since typescript function `getOrganizationEventTypeContext` return type is inferred automatically anyway. Very least i would remove export of this interface not to think whether or not its used somewhere else or not.